tasks/prime_workflow.py

```diff
@@ -0,0 +1,204 @@
+# -*- coding: utf-8 -*-
+import logging
+import random
+
+from locust import tag, task
+
+from .prime import PrimeTasks, SupportTasks
+from utils.constants import (
+    MOVE_TASK_ORDER,
+    MTO_SHIPMENT,
+    MTO_SERVICE_ITEM,
+    PAYMENT_REQUEST,
+)
+from utils.base import ImplementationError
+
+logger = logging.getLogger(__name__)
+
+
+@tag("workflow")
+class WorkflowTasks(PrimeTasks, SupportTasks):
+    """Workflow Task Set
+    Each task is a workflow from start to finish. Each workflow acts on only one MTO from start to finish.
+    There are multiple workflows for different scenarios such as a single shipment HHG move, or an HHG move with a SIT.
+    Multiple instances of the tasks will run during load testing and will interleave their hits to the server.
+    However they will run in sequence and should all complete without any errors.
+    """
+
+    # current_move is the move the whole task will be acting on
+    # all updates to the move and nested objects will be stored to this mto
+    current_move = None
+    # workflow_descriptor
+    workflow_descriptor = None
+
+    def on_start(self):
+        """
+        Runs on instantiation of each taskset.
+        We need to set up this task set with the list of tasks/workflows we want to run. If we don't define this, all
+        the tasks in PrimeTasks and SupportTasks will all be run independently and will likely have errors.
+        """
+        self.tasks = [self.hhg_move, self.fetch_mto_updates]
+
+    # WORKFLOWS
+    @tag("hhgMove")
+    @task
+    def hhg_move(self):
+        """
+        This is a basic HHG move workflow where we have one shipment and no additional requested service items.
+        To create a new flow, copy this function, but don't forget to add the workflow to the tasks array in the on_start function
+        """
+        self.workflow_title = "HHG Move Basic"
```

Comment:
I don't think this is ever used more than once, so I would delete this variable and just hardcode it into the log statement later on.

Reply:
Originally I had subfunctions with bits of the flow, and this was a way for each subfunction to log meaningful messages about what work they had done, like "HHG Move Basic - Completed creation and counseling". But since my ticket was only for a tiny piece of the flow, I removed the subfunctions. I think it will be useful once we have the whole flow?

Reply:
That's fair! Part of my cognitive dissonance with this was that it was different from the `workflow_descriptor` on the class level, but it makes more sense when they're the same thing